cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -14,6 +14,12 @@
 #
 ##################################################################
 
+body file control
+{
+        inputs => { "cfe_internal/enterprise/federation/federation.cf" };
+}
+
```

Comment:
I think one of these blank lines can be redacted.

Reply:
Yup